src/components/user/login/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import {
+  Grid, Row, Col, Alert,
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import { loginAPI } from '../../../store/actions/login';
+import { ROUTES } from '../../../constants';
+import LoginForm from './login-form';
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(fields) {
+    const { login } = this.props;
+    login(fields);
+  }
+
+  renderFormError() {
+    const { loginState } = this.props;
+    const { message } = loginState;
+
+    return message ? <Alert bsStyle="danger">{message}</Alert> : null;
+  }
+
+  render() {
+    const { loginState } = this.props;
+    const { statusCode, data } = loginState;
+    const { home } = ROUTES;
+    const isLoginSuccessful = statusCode === 200;
+    const loginFormProps = {
+      title: 'Login',
+      onSubmit: this.handleSubmit,
+      inputFields: [
+        { textInput: { name: 'email' } },
+        { textInput: { name: 'password' } },
+      ],
+    };
+
+    if (isLoginSuccessful) {
+      localStorage.setItem('token', data);
```

Comment:
"token" is a bit ambiguous. I would suggest renaming this "OS_AUTH_TOKEN" so that:
1. it's clear that it is used for authentication
2. It is namespaced with "OS" so it avoids potential conflicts with other libraries that may use the same token name